src/kmeans.jl

```diff
@@ -72,6 +92,8 @@ function _kmeans!{T<:AbstractFloat}(
     tol::Real,                      # in: tolerance of change at convergence 
     displevel::Int)                 # in: the level of display
 
+
+    
```

Comment:
Please remove the excess whitespace here and above. The change is unrelated to the PR.